src/components/Fields/PasswordField/PasswordField.tsx

```diff
@@ -51,6 +51,8 @@ const PasswordField: FC<PasswordFieldProps> = ({
     setFieldValue(field.name, event.target.value);
   };
 
+  console.log();
```

Comment:
I think we should get rid of it

Reply:
Actually don't remember that I did something with password field in this branch, that mystery how it appeared here